tests/integration/test_aws/test_basic.py

```diff
@@ -0,0 +1,113 @@
+import os
+
+import pytest
+from wazuh_testing import global_parameters
+from wazuh_testing.modules.aws import event_monitor
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import (
+    get_test_cases_data,
+    load_configuration_template,
+)
+from wazuh_testing.tools.monitoring import FileMonitor
+
+pytestmark = [pytest.mark.server]
+
+
+# Generic vars
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template', 'basic_test_module')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases', 'basic_test_module')
+local_internal_options = {'wazuh_modules.debug': '2', 'monitord.rotate_log': '0'}
+
+# ---------------------------------------------------- TEST_DEFAULTS ---------------------------------------------------
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_defaults.yaml')
+cases_path = os.path.join(TEST_CASES_PATH, 'cases_defaults.yaml')
+
+# Enabled test configurations
+configuration_parameters, configuration_metadata, case_ids = get_test_cases_data(cases_path)
+configurations = load_configuration_template(configurations_path, configuration_parameters, configuration_metadata)
+
+
+@pytest.mark.tier(level=0)
+@pytest.mark.parametrize('configuration, metadata', zip(configurations, configuration_metadata), ids=case_ids)
+def test_defaults(
+    configuration, metadata, load_wazuh_basic_configuration, set_wazuh_configuration, clean_s3_cloudtrail_db,
+    configure_local_internal_options_function, truncate_monitored_files, restart_wazuh_function, wazuh_log_monitor
+):
+    """
+    description: The module is invoked with the expected parameters and no error occurs.
+    test_phases:
+        - setup:
+            - Load Wazuh light configuration.
+            - Apply ossec.conf configuration changes according to the configuration template and use case.
+            - Apply custom settings in local_internal_options.conf.
+            - Truncate wazuh logs.
+            - Restart wazuh-manager service to apply configuration changes.
+        - test:
+            - Check in the ossec.log that a line has appeared calling the module with correct parameters.
+            - Check in the ossec.log that no errors occurs.
+        - tierdown:
```

Comment:
This is wrong in multiple tests.
```suggestion
        - teardown:
```